src/OpenLoco/src/Objects/TrainStationObject.cpp

```diff
@@ -148,14 +148,14 @@ namespace OpenLoco
         std::fill(std::begin(manualPower), std::end(manualPower), nullptr);
     }
 
-    std::vector<TrainStationObject::CargoOffset> TrainStationObject::getCargoOffsets(const uint8_t rotation, const uint8_t nibble) const
+    sfl::static_vector<TrainStationObject::CargoOffset, 16> TrainStationObject::getCargoOffsets(const uint8_t rotation, const uint8_t nibble) const
```

Comment:
I think we should have a constant somewhere for this,

Reply:
added